.github/workflows/juno-test.yml

```diff
@@ -29,6 +29,8 @@ jobs:
       - uses: Swatinem/rust-cache@v2
         with:
           workspaces: |
+            # do we need to add starknet/rust here?
+            # and what is there real benefit of using rust-cache?
```

Comment:
Yeah, I think starknet/rust should be added here as well. The benefit is faster CI builds. Juno's rust components rarely get's changed, so caching rust builds made sense back then.